src/main/java/edu/ucla/library/iiif/auth/handlers/ItemsHandler.java

```diff
@@ -49,24 +51,10 @@ public ItemsHandler(final Vertx aVertx) {
     @Override
     public void handle(final RoutingContext aContext) {
         final HttpServerRequest request = aContext.request();
-        final JsonArray requestData;
         final HttpServerResponse response = aContext.response() //
                 .putHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON.toString());
-        final JsonObject responseData;
 
-        try {
-            requestData = aContext.getBodyAsJsonArray();
-        } catch (final DecodeException details) {
-            responseData = new JsonObject() //
-                    .put(ResponseJsonKeys.ERROR, Error.INVALID_JSONARRAY) //
-                    .put(ResponseJsonKeys.MESSAGE, details.getMessage());
-            response.setStatusCode(HTTP.BAD_REQUEST).end(responseData.encodePrettily());
-
-            LOGGER.error(MessageCodes.AUTH_006, request.method(), request.absoluteURI(), details.getMessage());
-            return;
-        }
-
-        myDatabaseServiceProxy.setItems(requestData).onSuccess(result -> {
+        myDatabaseServiceProxy.setItems(aContext.body().asJsonArray()).onSuccess(result -> {
```

Comment:
Just as a defensive measure, it might be good to check that the body is available since it won't be if the body handler hasn't been set up correctly (if someone has overridden the default one with a broken one, for instance)?

Reply:
I don't think it's a practical concern because we have Vert.x OpenAPI setting all of that up for us, plus we're testing for this, so if someone overrides the default one with a broken one, ItemsHandlerIT::testPostItems will catch that.